docs/installation.rst

```diff
@@ -4,11 +4,76 @@
 Installation
 ============
 
-At the command line type::
+As mentioned in the :doc:`index`, HistomicsTK can be used both as a pure
+Python toolkit for algorithm development and as server-side Girder_ plugin
+for web-based analysis. Here, we describe how to install HistomicsTK for both
+these scenarios.
 
-    $ pip install histomicstk
+Installing HistomicsTK as a Python toolkit
+------------------------------------------
+
+HistomicsTK depends on large_image_ for reading large multi-resolution
+whole-slide images in a tiled fashion. Please see the Github repo of
+large_image to find out how to install it as a Python toolkit/package.
+
+HistomicsTK also leverages the functionality of a number of scientific python
+packages including numpy_, scipy_, scikit-image_, scikit-learn_,
+and pandas_. We recommend using anaconda to ease the cross-platform
+installation of these packages all of which are listed in
+`requirments_c_conda.txt <https://github.com/DigitalSlideArchive/HistomicsTK/blob/master/requirements_c_conda.txt>`__.
+
+Once large_image is installed, HistomicsTK can be installed as follows::
+
+    $ git clone git@github.com:DigitalSlideArchive/HistomicsTK.git
```

Comment:
I'd switch this to the `https` version instead of the `git` version (`https://github.com/DigitalSlideArchive/HistomicsTK.git`).  In my test VM, the `git` version failed, while the `https` version worked just fine.